core/crypto/secp256k1_types.hpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CRYPTO_SECP256K1_TYPES_HPP
+#define KAGOME_CRYPTO_SECP256K1_TYPES_HPP
+
+#include "common/blob.hpp"
+
+namespace kagome::crypto {
+  namespace secp256k1 {
+    static constexpr size_t kUncompressedPublicKeySize = 65u;
+    static constexpr size_t kCompressedPublicKeySize = 33u;
+    static constexpr size_t kCompactSignatureSize = 65u;
+  }  // namespace secp256k1
+
+  /**
+   * uncompressed form of public key
+   */
+  using Secp256k1CompressedPublicKey =
```

Comment:
Think about move types into secp256k1 namespace and remove 'Secp256k1-' prefixes. I guess this is redundant.

Reply:
reafactored